ui/page/send/send_destination.go

```diff
@@ -47,8 +48,14 @@ func newSendDestination(l *load.Load) *destination {
 		{Text: values.String(values.StrWallets)},
 	})
 
+	dst.initDestinationWalletSelector(selectedWallet)
+
+	return dst
+}
+
+func (dst *destination) initDestinationWalletSelector(selectedWallet sharedW.Asset) {
```

Comment:
What you need here is the asset type, not the selected wallet. The real selected wallet should be gotten from the `dst.destinationWalletSelector` via `dst.destinationWalletSelector.SelectedWallet()`.